{"author":{"id":"ff90e8c1032712f76e3c119e253ae3a84614b47da224d1a3450b4f469b0523a9"},"ops":[{"type":3,"timestamp":1614884071,"nonce":"575iobQ3MCQXEiDUQmFPPvpqmlU=","metadata":{"github-id":"MDEyOklzc3VlQ29tbWVudDc5MDg0ODI2Mw==","github-url":"https://github.com/svpcom/wfb-ng/issues/118#issuecomment-790848263"},"message":"But now drone got some packages from GS:\n```\nMar 04 17:41:18 raspberrypi python[1164]: 2021-03-04 17:41:18+0000 [-] tunnel rx: unable to decrypt packet #0x3300\nMar 04 17:41:18 raspberrypi python[1164]: 2021-03-04 17:41:18+0000 [-] tunnel rx: unable to decrypt packet #0x3301\n```\nbut it unable to decrypt them. Check that drone.key corresponds to gs.key\nAlso check that NetworkManager (or other tools like wpa_supplicant) are disabled for `wlan1`. Do `ps uaxwwww | grep wlan1` it should show nothing","files":null}]}